controller/tests/test_plan_controller.py

```diff
@@ -36,23 +19,22 @@ def test_close_plan_controller(self):
                     description='Test emergency plan',
                     geographical_area='',
                     camps=[Camp(name='TestCamp')])
-        close_plan(plan)
-        self.assertEqual(True, plan.close_plan_status)
-        self.setUp()
+        pc.close_plan(plan)
+        self.assertEqual(True, plan.is_closed)
 
-    def test_no_close_date(self):
+    def test_initialised_plan_no_close_date(self):
         """
-        Test that close date is not set during plan creation.
+        Test plan closure date is set to today's date when plan is closed.
```

Comment:
This comment looks incorrect. The old version should be the correct one

Reply:
I missed that, thanks. I've corrected it now!